Components/Model/Order.php

```diff
@@ -101,11 +101,15 @@ public function loadData(\Shopware\Models\Order\Order $order)
 	{
 		$this->_orderNumber = $order->getNumber();
 		$this->_createdDate = $order->getOrderTime()->format('Y-m-d');
-
-		$this->_paymentProvider = $order->getPayment()->getName();
-		$paymentPlugin = $order->getPayment()->getPlugin();
-		if (!is_null($paymentPlugin)) {
-			$this->_paymentProvider .= sprintf(' [%s]', $paymentPlugin->getVersion());
+		$payment = $order->getPayment();
+		try {
+			$this->_paymentProvider = $payment->getName();
+			$paymentPlugin = $payment->getPlugin();
+			if (!is_null($paymentPlugin)) {
```

Comment:
You could also check that  `$paymentPlugin->getVersion()` is not empty. Currently this could lead to a payment provider like this "provider []"